lib/std/zig/system/NativePaths.zig

```diff
@@ -37,6 +37,10 @@ pub fn detect(arena: Allocator, native_target: std.Target) !NativePaths {
                 if (mem.startsWith(u8, word, "-frandom-seed=")) {
                     continue;
                 }
+                if (mem.eql(u8, word, "-idirafter")) {
+                    _ = it.next().?;
```

Comment:
`?` will incorrectly crash if missing arg. also the information must not be discarded.